### DISCUSSION TITLE
feat[github.com] repository orginization/folders

### DISCUSSION THREAD

**Discussion by jerodg** [community/community#27537] [Other Feature Feedback, Questions, & Ideas]
After building quite a few repositories over the years I find it a pain to navigate through my repos. Many of them could easily be put under a folder or perhaps tagged so that they can be viewed in a folder-like manner on the website. This would make it much easier to navigate a large number of repos. Honestly, any organizational system would be an improvement over the current system.

**Comment by DenverCoder1** [comment]
This may not be an ideal solution for you, but [GitHub Lists](https://github.blog/changelog/2021-12-09-lists-are-now-available-as-a-public-beta/) are a new feature (in Beta) on the stars page.

You would be able to add your repos to lists using the star menu.

![image](https://user-images.githubusercontent.com/20955511/181657165-b3724118-7c2a-434e-a01d-1d45516a7861.png)

This does mean the repositories would have to be marked as "starred" by you, I believe.

**Reply by radar-blip** [reply]
Yes, starred by you. And put into some kind of catagorization.

**Comment by jfullstackdev** [comment]
you have the Lists and you have the Topics, for example

![image](https://user-images.githubusercontent.com/47092464/181675316-e994d729-0692-4f56-9690-3b22f88b389d.png)

![image](https://user-images.githubusercontent.com/47092464/181675356-b16ed849-ca77-4190-90c5-c6fdbb3ab892.png)

if I click `jdevstatic-java`, this is what I will see 

![image](https://user-images.githubusercontent.com/47092464/181675419-def4834d-3a7b-4ebb-b8bb-755bd5c4c379.png)

it's like a directory by now

**Comment by jerodg** [comment]
There is a limit to pins and from what I assume pins are meant to showcase
a few projects on your profile page.

I'm only 10yrs into my career. I expect by the end to have 10s of thousands
of repos. Without organization of some kind it will be an impossible task
to locate anything.

**Comment by jfullstackdev** [comment]
there is the external service for that so you can pin more, but I don't know how many, maybe you can ask there it too, but it will provide more pins than the default ones, 

https://github.com/anuraghazra/github-readme-stats#github-extra-pins
